forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -223,6 +223,10 @@ pub struct Command {
     /// The output format to use; possible values: default, raw
     #[clap(long, default_value = "default")]
     pub output: OutputFormat,
+
+    /// Output call receipts
+    #[clap(long, alias = "r")]
```

Comment:
Oh this should be done through `short`

```suggestion
    #[clap(long, short = "r")]
```

and you should be able to have alias as well

```suggestion
    #[clap(long, short = "r", alias = "receipts")]
```

https://docs.rs/clap/latest/clap/struct.Arg.html#method.alias

As per the docs alias is for providing alternatives to the long format:


> Add an alias, which functions as a hidden long flag.

Reply:
oversight; addressed: https://github.com/FuelLabs/sway/pull/6968/commits/70614355cd904311dc5fa5c97270a39c9d2f87e2